dll/win32/browseui/res/autocompletehistory.rgs

```diff
@@ -0,0 +1,13 @@
+HKCR
+{
+	NoRemove CLSID
+	{
+		ForceRemove {00BB2764-6A77-11D0-A535-00C04FD7D062} = s 'Microsoft History AutoComplete List'
```

Comment:
Is there anything that depends on the `Microsoft` here?
If not, I would prefer it to not include Microsoft in the name.

Reply:
If you want to change it, maybe replacing "Microsoft" by "ReactOS"?

Reply:
OK.